static/js/ui.js

```diff
@@ -472,7 +472,7 @@ $(function () {
         timerender.set_full_datetime(message, time_elem);
     });
 
-    $('#streams_inline_cog').tooltip({ placement: 'left',
+    $('.streams_inline_cog').tooltip({ placement: 'left',
```

Comment:
No need to make `streams-inline-cog` a CSS class. You can use an `#id` selector (since it's a unique element)